pkg/operator/baremetal_pod.go

```diff
@@ -235,13 +238,40 @@ func createMetal3PasswordSecrets(client coreclientv1.SecretsGetter, config *Oper
 	return nil
 }
 
+// Return false on error or if "baremetal.openshift.io/owned" annotation set
+func checkMetal3DeploymentMAOOwned(client appsclientv1.DeploymentsGetter, config *OperatorConfig) (bool, error) {
+	existing, err := client.Deployments(config.TargetNamespace).Get(context.Background(), "metal3", metav1.GetOptions{})
```

Comment:
Should the `"metal3"` here be `baremetalDeploymentName`?

Reply:
Yes. Made that change.